src/components/Chip/Chip.tsx

```diff
@@ -20,16 +20,21 @@ function Chip({ label, selected = false, disabled = false, onClick = () => {} }:
     'cursor-not-allowed': disabled,
     'cursor-pointer': !disabled,
   });
-  
-return (
-    <button
-      type="button"
+
+  const handleKeyDown = (event: React.KeyboardEvent) => {
+    if (event.key === "Enter") onClick();
+  };
```

Comment:
why are we doing this?

Reply:
@robdembitel   because of this listing error  Visible, non-interactive elements with click handlers must have at least one keyboard listener

Reply:
this does not feel right. what does the linting say exactly?